app/src/utils/utils.ts

```diff
@@ -28,22 +42,69 @@ export function isValidAddress(val: string | undefined) {
   return val && isAddress(val);
 }
 
-// Expects a unix timestamp and will return a human readable message of how far in the past/future it is
-export function daysAgo(val = 0) {
-  // Use a formatter to establish "in 10 days" vs "10 days ago"
-  const formatter = new Intl.RelativeTimeFormat();
-  // Number of days since now
-  const deltaDays = (val * 1000 - Date.now()) / (1000 * 3600 * 24);
+// --- Grants + Cart ---
+const CART_KEY = 'cart';
+const DEFAULT_CONTRIBUTION_TOKEN_ADDRESS = '0x6B175474E89094C44Da98b954EedeAC495271d0F'; // DAI
+const DEFAULT_CONTRIBUTION_AMOUNT = 5; // this is converted to a parsed BigNumber at checkout
 
-  // Format "days ago" as string
-  return formatter.format(Math.round(deltaDays), 'days');
+// Loads cart data
+export function loadCart(): CartItemOptions[] {
```

Comment:
This file is starting to grow in size -> would it make sense to move all of this into `app/src/utils/cart.js`

Reply:
Pulling them out into `cart.js` seems like a good idea

Reply:
Done in 97fc542e1c60e1958e367ff22e3c96ef23d156de